# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fastcgi/gcc44.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- a/include/fcgio.h	2009-01-28 11:28:10.000000000 -0500
+++ b/include/fcgio.h	2009-01-28 11:30:15.000000000 -0500
@@ -31,6 +31,7 @@
 #define FCGIO_H
 
 #include <iostream>
+#include <cstdio>
 
 #include "fcgiapp.h"
 
